Makefile

```diff
@@ -0,0 +1,25 @@
+# Allow linting on specific filepaths if needed
+FILEPATH :=
+
+# Issues to ignore while linting
+LINT_IGNORE :=
+
+.PHONY: run
+run:
+	@python3 scripts/aggregate_can_data.py \
```

Comment:
This makefile might come in handy to connect everything. However, I don't think we shouldn't be running these scripts.